src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_help.py

```diff
@@ -0,0 +1,379 @@
+    examples:
+        - name: Creates Authorizationrule
+          text: az eventhubs namespace authorization-rule create --resource-group myresourcegroup --namespace-name mynamespace --name myauthorule --rights Send Listen
+"""
+
+helps['eventhubs namespace authorization-rule update'] = """
+    type: command
+    short-summary: Updates Authorizationrule for the given Namespace
+    examples:
+        - name: Updates Authorizationrule
+          text: az eventhubs namespace authorization-rule update --resource-group myresourcegroup --namespace-name mynamespace --name myauthorule --rights Send
+"""
+
+helps['eventhubs namespace authorization-rule show'] = """
+    type: command
+    short-summary: Shows the details of Authorizationrule
+    examples:
+        - name: Shows the details of Authorizationrule
+          text: az eventhubs namespace authorization-rule show --resource-group myresourcegroup --namespace-name mynamespace --name myauthorule
+"""
+
+helps['eventhubs namespace authorization-rule list'] = """
+    type: command
+    short-summary: Shows the list of Authorizationrule by Namespace
+    examples:
+        - name: Shows the list of Authorizationrule by Namespace
+          text: az eventhubs namespace authorization-rule show --resource-group myresourcegroup --namespace-name mynamespace
+"""
+
+helps['eventhubs namespace authorization-rule keys list'] = """
+    type: command
+    short-summary: Shows the connection strings for namespace
+    examples:
+        - name: Shows the connection strings of Authorizationrule for the namespace.
+          text: az eventhubs namespace authorization-rule list-keys --resource-group myresourcegroup --namespace-name mynamespace --name myauthorule
+"""
+
+helps['eventhubs namespace authorization-rule keys renew'] = """
+    type: command
+    short-summary: Regenerate the connection strings of Authorizationrule for the namespace.
+    examples:
+        - name: Regenerate the connection strings of Authorizationrule for the namespace.
+          text: az eventhubs namespace authorization-rule regenerate-keys --resource-group myresourcegroup
+           --namespace-name mynamespace --name myauthorule --key PrimaryKey
+"""
+
+helps['eventhubs namespace authorization-rule delete'] = """
+    type: command
+    short-summary: Deletes the Authorizationrule of the namespace.
+    examples:
+        - name: Deletes the Authorizationrule of the namespace.
+          text: az eventhubs namespace authorization-rule delete --resource-group myresourcegroup --namespace-name mynamespace --name myauthorule
+"""
+
+helps['eventhubs eventhub create'] = """
+    type: command
+    short-summary: Creates the Event Hubs Eventhub
+    examples:
+        - name: Create a new Eventhub.
+          text: az eventhubs eventhub create --resource-group myresourcegroup --namespace-name mynamespace --name myeventhub --message-retention-in-days 4 ---partition-count 15
```

Comment:
This and several other examples need to be updated for changed names. Here, you need to remove "-in-days".

Reply:
resolved